### PR TITLE
Fix links in Full version

### PR DESCRIPTION
**Pull request by Nanush7**
Fixes the links in the full version. Given that this is now `/regulations/full`, `./full` was pointing to `/regulations/full/full`.

### CODE REVIEW

File: wrc/codegen/cghtml.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def __init__(self, versionhash, language, pdf, merged=False):
         super(WCADocumentHtml, self).__init__(str)
         self.regset = set()
         if merged:
-            self.urls = {'regulations': './full', 'guidelines': './full', 'pdf': pdf}
+            self.urls = {'regulations': './', 'guidelines': './', 'pdf': f'../{pdf}'}
```
Comment (gregorbg) [inline wrc/codegen/cghtml.py:97]: why does the PDF need the `../` backwards navigation?
That's not happening in the "non-merged" documents, why does the merged version suddenly need it?

Comment (Nanush7) [reply wrc/codegen/cghtml.py:97]: @gregorbg having both PDFs sounds good to you?

**Comment by Nanush7** [comment]
I just reverted the pdf path change. After my exams I can submit a PR to generate the merged PDF.

We are receiving a lot of message about broken links, so it would be good to merge this as soon as possible. :)

**Review by gregorbg — APPROVED** [review]
Current diff looks fine for me, if that works as intended then let's go ahead and deploy

**Pull request merged**